yt_dlp/extractor/vk.py

```diff
@@ -87,6 +87,22 @@ def _download_payload(self, path, video_id, data, fatal=True):
             raise ExtractorError(clean_html(payload[0][1:-1]), expected=True)
         return payload
 
+    def _get_access_token(self, video_id):
+        try:
+            data = self._download_json(
+                'https://login.vk.com/?act=get_anonym_token&client_secret=o557NLIkAErNhakXrQ7A'
+                '&client_id=52461373&scopes=audio_anonymous%2Cvideo_anonymous%2Cphotos_anonymous%2Cprofile_anonymous',
```

Comment:
don't split URLs into multiple lines like this

use a `query` param dict to keep the URL line short and to keep this readable/maintainable